Content/Maatregelen/M19/ICTU.md

```diff
@@ -1,3 +1,7 @@
 #### ICTU
 
 ICTU ondersteunt dit met Docker en/of virtuele machines (VM) en een VLAN per project. Een nieuwe afgeschermde digitale omgeving is binnen een werkweek na aanvraag beschikbaar.
+
+Binnen de projectomgeving bepaalt het projectteam zelf de gewenste configuratie.
+Veranderingen aan de omgeving komen alleen in overleg met de centrale beheergroep tot stand.
```

Comment:
Wat voor veranderingen gaat het hier over? Veranderingen anders dan aan de configuratie? Wat die bepaalt het projectteam zelf, staat in de vorige regel.